Packs/HYASInsight/Integrations/HYASInsight/HYASInsight.yml

```
@@ -0,0 +1,393 @@
+      description: The international domain name
+      type: String
+    - contextPath: HYAS.WHOIS.nameserver
+      description: nameserver
+      type: Unknown
+    - contextPath: HYAS.WHOIS.phone.phone
+      description: The phone number registrant contact in e164 format
+      type: String
+    - contextPath: HYAS.WHOIS.phone.phone_info.carrier
+      description: Phone number carrier
+      type: String
+    - contextPath: HYAS.WHOIS.phone.phone_info.country
+      description: Phone number country
+      type: String
+    - contextPath: HYAS.WHOIS.phone.phone_info.geo
+      description: Phone number geo. Can be city, province, region or country
+      type: String
+    - contextPath: HYAS.WHOIS.privacy_punch
+      description: True if this record has additional information bypassing privacy
+        protect
+      type: Boolean
+    - contextPath: HYAS.WHOIS.registrar
+      description: The domain registrar
+      type: String
+  - arguments:
+    - default: false
+      description: Domain value to query
+      isArray: false
+      name: domain
+      required: true
+      secret: false
+    deprecated: false
+    description: Returns WHOIS Current records for the provided indicator value.
+    execution: false
+    name: hyas-get-whois-current-records-by-domain
+    outputs:
+    - contextPath: HYAS.WHOISCurrent.items.abuse_emails
+      description: abuse emails
+      type: Unknown
+    - contextPath: HYAS.WHOISCurrent.items.address
+      description: address
+      type: Unknown
+    - contextPath: HYAS.WHOISCurrent.items.city
+      description: city
+      type: Unknown
+    - contextPath: HYAS.WHOISCurrent.items.country
+      description: country
+      type: Unknown
+    - contextPath: HYAS.WHOISCurrent.items.domain
+      description: The domain of the registrant
+      type: String
+    - contextPath: HYAS.WHOISCurrent.items.domain_2tld
+      description: The second-level domain of the registrant
+      type: String
+    - contextPath: HYAS.WHOISCurrent.items.domain_created_datetime
+      description: The date and time when the whois record was created
+      type: Date
+    - contextPath: HYAS.WHOISCurrent.items.domain_expires_datetime
+      description: The date and time when the whois record expires
+      type: Date
+    - contextPath: HYAS.WHOISCurrent.items.domain_updated_datetime
+      description: The date and time when the whois record was last updated
+      type: Date
+    - contextPath: HYAS.WHOISCurrent.items.email
+      description: email
+      type: Unknown
+    - contextPath: HYAS.WHOISCurrent.items.idn_name
+      description: The international domain name
+      type: String
+    - contextPath: HYAS.WHOISCurrent.items.nameserver
+      description: nameserver
+      type: Unknown
+    - contextPath: HYAS.WHOISCurrent.items.organization
+      description: organization
+      type: Unknown
+    - contextPath: HYAS.WHOISCurrent.items.phone
+      description: The phone number
+      type: Unknown
```

I think you are missing outputs here for phone because it is an array. I guess it is like the WHOIS above . I added a suggestion that you can commit if indeed i am correct and you missed it
```suggestion
    - contextPath: HYAS.WHOISCurrent.phone.phone
      description: The phone number registrant contact in e164 format
      type: String
    - contextPath: HYAS.WHOISCurrent.phone.phone_info.carrier
      description: Phone number carrier
      type: String
    - contextPath: HYAS.WHOISCurrent.phone.phone_info.country
      description: Phone number country
      type: String
    - contextPath: HYAS.WHOISCurrent.phone.phone_info.geo
      description: Phone number geo. Can be city, province, region or country
      type: String
```